src/main/kotlin/lotto/model/RandomLottoTicketStorage.kt

```diff
@@ -4,14 +4,19 @@ import kotlin.random.Random
 
 object RandomLottoTicketStorage : LottoTicketStorage {
 
-    private const val LOTTO_TICKET_SIZE: Int = LottoTicket.SIZE
+    private const val LOTTO_SIZE: Int = Lotto.SIZE
     private val LOTTO_NUMBERS: Set<LottoNumber> = LottoNumber.ALL
 
-    override val lottoTicket: LottoTicket
-        get() {
-            return LOTTO_NUMBERS.shuffled(Random.Default)
-                .take(LOTTO_TICKET_SIZE)
+    override fun hasCountLessThan(count: Int): Boolean {
```

Comment:
```suggestion
    override infix fun hasCountLessThan(count: Int): Boolean {
```
infix 가 누락되다보니, 이 함수가 infix 인지 아닌지 인지하기가 어렵네요. 🤔 
infix를 붙여보면 어떨까요?

Reply:
IDE 에서 함수를 자동으로 오버라이드를 했더니 `infix` 가 누락됐네요 😅
이 부분은 수정하도록 하겠습니다!!